aspnetcore/fundamentals/routing.md

```diff
@@ -30,7 +30,7 @@ Routing uses *routes* (implementations of `IRouter`) to:
 
 * generate URLs used in responses
 
-Generally an app has a single collection of routes. The route collection is processed in order. Requests look for a match in the route collection by . Responses use routing to generate URLs.
+Generally an app has a single collection of routes. The route collection is processed in order. Requests look for a match in the route collection by . Responses use routing to generate URLs (for example, for redirection).
```

Comment:
Ooops. Missed this comment. I've edited a bit more in the meantime. Let me know what you think. I can roll back the changes and fix as suggested instead.